DockerDevSetupGuideForMacOS.md

```diff
@@ -162,9 +162,11 @@ Reference: https://github.com/rbenv/rbenv
 # Open DRLS REMS as VsCode workspace 		
 The REMS repository contains the **REMS.code-workspace** file, which can be used to open the above project structure as a multi-root VS Code workspace. To open this workspace, select *File* > *Open Workspace from File...* and navigate to <drls-root>/REMS/REMS.code-workspace. In this workspace configuration, the CDS-Library embedded within CRD is opened as a seperate root for an easier development experience.	
 
-The Source Control Tab can be used to easily track changes during the devlopement process and perform git actions, with each root of the workspace having its own source control header. See: https://code.visualstudio.com/docs/editor/versioncontrol	
+The Debugger Tab has various debugging configurations and can be used to easily debug any errors that come up during development. That said, when debugging it is reccomeneded to use the **Debug All REMS Applications (Dokcer) (workspace)** debugger as this is a compund debugger that combines the Docker debugger for all servers and applications in this workspace, simply set a breakpoint anywhere in the code base and debug away. For more information on VsCode debugging see: https://code.visualstudio.com/docs/editor/debugging 
```

Comment:
```suggestion
The Debugger Tab has various debugging configurations and can be used to easily debug any errors that come up during development. That said, when debugging it is recommended to use the **Debug All REMS Applications (Docker) (workspace)** debugger as this is a compound debugger that combines the Docker debugger for all servers and applications in this workspace, simply set a breakpoint anywhere in the code base and debug away. For more information on VS Code debugging see: https://code.visualstudio.com/docs/editor/debugging 
```

Reply:
Not necessary, but the second sentence here is pretty long and uses the word debug / debugger a lot. I had to re-read it twice, which admittedly might just be because I am tired. Maybe shorten or break up to something like:

> It is recommended to use the **Debug All REMS Applications [Docker] [workspace]** debugger. This is a compound debugger that combines context of all servers and applications in this workspace. You may simply set a breakpoint anywhere in the code-base and debug from there.